vortex-ipc/src/messages/decoder.rs

```diff
@@ -30,9 +30,10 @@ pub enum DecoderMessage {
 pub struct ArrayParts {
     row_count: usize,
     // Typed as fb::Array
-    array_flatbuffer: Buffer,
+    // FIXME(ngates): use ConstAlignedArray aliased to FlatBuffer
```

Comment:
should probably file an issue for this instead of writing repetitive FIXMEs to this effect in a bunch of places...